### PR TITLE
Origin/d gs br105

### PR DESCRIPTION
**Pull request by Roel-Barro**
new task

### CODE REVIEW

File: one.cfm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<hr/>
+this is a new file from new branch
+<hr/>
+<cfinclude  template="two.cfm">
```
Comment (cattool) [inline one.cfm:5]: add more br

Comment (Roel-Barro) [reply one.cfm:5]: what line

Comment (cattool) [reply one.cfm:5]: before line 4

**Review by cattool — APPROVED** [review]
all good

**Pull request merged**